examples/express-sample/views/index.ejs

```diff
@@ -11,6 +11,9 @@
     <div>
       <a href="/send">Send a custom error</a>
     </div>
+    <div>
+      <a href="/skip">Send an error that will be skipped</a>
```

Comment:
Maybe this instead - 
```suggestion
      <a href="/skip">Send an error that uses onBeforeSend and doesn't send to Raygun</a>
```

Reply:
That won't be correct, because `onBeforeSend` is used by all attempts to send an error report.

Reply:
I think I read this wrong! Will resolve this thread